coding-intro/coding-intro-core/core-concepts/do-it-yourself.md

```diff
@@ -0,0 +1,36 @@
+---
+author: abdullah
+tags:
+  - coding
+type: normal
+category: coding
+
+setupCode:
+  startingPoint: "# Who did you meet today?. \n name = '???' \n age = ??? \n print(age)  # Print age to the screen as it is. \n print(name/2)  # Print name to screen after dividing by 2!? \n"
+---
+# Do it yourself!
+---
+## Content
+
+Welcome to your first coding task.
+
+Who's the last person you met? Do you remember their age?
+
+In the following task, you will store that person's name and age in *variables*. You will also print them to the screen, using **Python**!
+
+After you have stored the variables correctly, the code will still show an error. Something's wrong in one of the `print` functions. 
+
+Your job is to fill out the missing part of the code and fix the existing one. You got this!
+
+Recall the following concepts:
+
+- Double-quotes (`""`) are necessary to store text data.
+- We cannot perform numeric operations on text.
+
+Don't worry too much about understanding every part of the code; see if you can solve it as is, but things will become clearer as you go along.
+
+
+> 💡 The [comments](https://enki.com/glossary/general/comment) might help you find what's wrong.
+> 💡 The guidelines above are just suggestions. Feel free to include other concepts in your solution as you see fit. The implementation is up to you.
+> 🤓 Happy learning! Open the playground and start coding!
```

Comment:
```suggestion
> 💡 The [comments](https://enki.com/glossary/general/comment) might help you find what's wrong.

> 💡 The guidelines above are just suggestions. Feel free to include other concepts in your solution as you see fit. The implementation is up to you.

> 🤓 Happy learning! Open the playground and start coding!
```

Reply:
I would separate them like this and then test them in the app again. I think this time they will display properly

Reply:
I will try it out right now